cmd/cluster/resize/infra_node.go

```diff
@@ -32,7 +32,7 @@ import (
 const (
 	twentyMinuteTimeout                   = 20 * time.Minute
 	twentySecondIncrement                 = 20 * time.Second
-	resizedInfraNodeServiceLogTemplate    = "https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/infranode_resized_auto.json"
+	resizedInfraNodeServiceLogTemplate    = "https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/infranode_resized.json"
 	GCPresizedInfraNodeServiceLogTemplate = "https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/gcp/GCP_infranode_resized_auto.json"
```

Comment:
/hold
Waiting until the service log are properly updated

Reply:
Done via https://github.com/openshift/managed-notifications/pull/327